### DISCUSSION TITLE
Diagnostic data present in Browser

### DISCUSSION THREAD

**Discussion by mostlyobvious** [RailsEventStore/rails_event_store#1881] [Ideas]
A subpage in Browser could summarize useful information in terms of diagnostics:
- RES release version (already in the footer but can be moved or duplicated)
- total number of events
- total number of streams
- repository adapter (optionally RDBMS used as well)

I'd also consider letting one voluntarily and anonymously submit this particular data to maintainers (google form or similar) to have better insight what a typical RES deployment looks like. Obviously being extremely precise what is sent and what code is responsible for that.

**Comment by mostlyobvious** [comment]
Possibly knowing a platform (Heroku or AWS) could be helpful for us to optimize overall RES experience.

**Comment by mostlyobvious** [comment]
It could be useful to inspect database schema (if RDBMS present)

**Comment by swistak35** [comment]
I'm not sure what information we could retrieve, having in consideration that we would like to have some useful data from maintainers perspective and don't loose trust by executing doubtful sql/shell scripts:
* RES version (easy and not problematic)
* Repository name (rather easy, at least sharing the class name)
* RDBMS used, in case of `RailsEventStoreActiveRecord` (probably easy, by looking at adapter used by active record?)
* infrastructure provider used (Heroku probably can be detected by looking at ENVs [ 
https://devcenter.heroku.com/articles/dyno-metadata ]: that's probably ok; EngineYard looks like could be identified by looking at binaries in `/usr/local/bin`, but this is IMO something which I would not like to see in event store library)
* whether the instance of event store has `LinkByCausationId`, `LinkByCorrelationId` or `LinkByEventType` in global handlers
* information about which of dispatchers, from the ones we are providing, are used (especially whether people are using ActiveJob dispatcher or not) -- we probably don't care about custom ones
* similarly with mappers, especially EncryptionMapper and Protobuf
